src/main/webapp/js/instructorCoursesAjax.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function(){
+    var ajaxRequest = function(e) {
+        e.preventDefault();
+        var formData = $(this).serialize();
+        $.ajax({
+            type: 'POST',
+            cache: false,
+            url: $(this).attr('action') + '?' + formData,
+            beforeSend: function() {
+                $('#coursesList').html(
+                    '<img height="75" width="75" class="margin-center-horizontal" src="/images/ajax-preload.gif"/>'
+                );
+            },
+            error: function() {
+
```

Comment:
@damithc So should I create a reload button or just show a message telling the user to reload the page ?